src/apps/create-patron-user-info/CreatePatron.dev.tsx

```diff
@@ -0,0 +1,134 @@
+import React from "react";
+import { ComponentMeta, ComponentStory } from "@storybook/react";
+import CreatePatron from "./CreatePatron.entry";
+import serviceUrlArgs from "../../core/storybook/serviceUrlArgs";
+import pincodeArgs from "../../core/storybook/pincodeArgs";
+
+export default {
+  title: "Apps / Create patron",
+  component: CreatePatron,
+  argTypes: {
+    ...serviceUrlArgs,
+    ...pincodeArgs,
+    blacklistedPickupBranchesConfig: {
+      defaultValue: "FBS-751032,FBS-751031,FBS-751009,FBS-751027,FBS-751024",
```

Comment:
Don't known these values all (to me known) agency id's (ISIL number) starts with `DK-xxxxxx`?

Reply:
I copy pasted, used throughout the system and I am actually not sure it makes a difference as it seems to work in the drupal setup :)